code/datums/materials/alloys.dm

```diff
@@ -0,0 +1,124 @@
+/** Materials made from other materials.
+  */
+/datum/material/alloy
+	name = "alloy"
+	desc = ""
+	/// The materials this alloy is made from weighted by their ratios.
+	var/list/composition = null
+	/// Breakdown flags required to reduce this alloy to its component materials.
+	var/req_breakdown_flags = BREAKDOWN_ALLOYS
+
+/datum/material/alloy/return_composition(amount=1, breakdown_flags)
+	if(req_breakdown_flags & !(breakdown_flags & req_breakdown_flags))
+		return ..()
+
+	. = list()
+	var/list/cached_comp = composition
+	for(var/comp_mat in cached_comp)
+		var/datum/material/component_material = SSmaterials.GetMaterialRef(comp_mat)
+		var/list/component_composition = component_material.return_composition(cached_comp[comp_mat], breakdown_flags)
+		for(var/comp_comp_mat in component_composition)
+			.[comp_comp_mat] += component_composition[comp_comp_mat] * amount
+
+
+/** Plasteel
+  *
+  * An alloy of iron and plasma.
+  */
+/datum/material/alloy/plasteel
```

Comment:
As mentioned in other comments, a lot of these materials are FAR too strong in the current material balance, Value, strength and armor ratings need to go down by a lot, Most materials should not be a perfect solution to everything, and instead have buffs and debuffs. If you want plasteel to for example be as strong as it is, wearing plasteel objects should slow you down etc. Feel free to add new components/elements for that (Optionally in a separate PR). However, if things like that aren't added these stats are simple too high, we can talk about this more in-depth on Discord if you'd like.

Reply:
I've nerfed them a bit, they are probably still way too strong though.

Reply:
I think so too, unless making alloys becomes a more difficult process I don't think they should be much more powerful than the materials theyre made of.

Reply:
Metallurgy can wait for another time. Any advice for what else to nerf?